### PR TITLE
Coordinator can process request to disable exporter 

### LINKED ISSUE #18420: Coordinator can handle exporter disable request
Similar to any topology change request like scaling, a coordinator can accept a request to disable an exporter, generates necessary operations for the change and starts a configuration change.

### PR DESCRIPTION
**Pull request by deepthidevaki**
## Description

A coordinator broker (which is usually the broker with the lowest id in the cluster) can accept a request to disable an exporter. When the request is received, it is translated to a sequence of operations which is then added to the cluster change plan, similar to how other topology change requests like scaling are handled.

This PR 
1. Defines the proto message for the request
2. Implements the request logic in the receiving broker.
3. Implements the sender that can send the request to the coordinator.

## Related issues

closes #18420

### CODE REVIEW

File: zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/api/ClusterConfigurationRequestTopics.java [hunk 1]
```diff
@@ -16,7 +16,8 @@ public enum ClusterConfigurationRequestTopics {
   SCALE_MEMBERS("topology-member-scale"),
   QUERY_TOPOLOGY("topology-query"),
   CANCEL_CHANGE("topology-change-cancel"),
-  FORCE_SCALE_DOWN("topology-force-scale-down");
+  FORCE_SCALE_DOWN("topology-force-scale-down"),
+  DISABLE_EXPORTER("topology-exporter-disable");
```
Comment (npepinpe) [inline zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/api/ClusterConfigurationRequestTopics.java:20]: :sweat_smile: "topology"? I get the why though :smile:

**Review by npepinpe — APPROVED** [review]
:question: For us to write a QA/E2E test, we're only missing https://github.com/camunda/zeebe/issues/18419 right?

File: zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/api/ClusterConfigurationRequestTopics.java (hunk 1 above)
Comment (deepthidevaki) [reply zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/api/ClusterConfigurationRequestTopics.java:20]: :see_no_evil: I feel it weird to only name this one differently. That said, I would like to rename all of it, but have to do it in a backward compatible way. May be one version that can listens to both topic and the next version which removes the old ones :thinking:

**Comment by deepthidevaki** [comment]
> ❓ For us to write a QA/E2E test, we're only missing #18419 right?

Also the gateway endpoints #18421

**Pull request merged**